web/src/locales/da.json

```diff
@@ -1,9 +1,85 @@
 {
+  "info": {
+    "title": "Om appen",
+    "text": "Electricity Maps-appen giver realtidsindsigt, på web og mobil, for borgere, energieksperter, NGO'er og beslutningstagere, der fremmer en dybere forståelse af energiomstillingens udfordringer.",
+    "open-source-text": "Denne app er <a href='{{link}}' target='_blank'>Open Source</a> og drevet af fællesskabet.",
+    "version": "App-version: {{version}}",
+    "share-app": "Del appen:"
+  },
+  "settings-modal": {
+    "title": "Indstillinger"
+  },
   "ranking-panel": {
     "title": "Klimapåvirkning per region",
     "subtitle": "Sorteret efter CO₂-intensitet fra elforbruget (gCO₂eq/kWh)",
     "search": "Søg efter region"
   },
+  "button": {
+    "reddit": "Join os på r/electricitymaps",
+    "github": "Bidrag på GitHub",
+    "twitter": "Følg os på Twitter",
+    "twitter-share": "Del på Twitter",
+    "slack": "Join os på vores Slack-fællesskab",
+    "linkedin": "Følg os på LinkedIn",
+    "linkedin-share": "Del på LinkedIn",
+    "facebook": "Følg os på Facebook",
+    "facebook-share": "Del på Facebook",
+    "feedback": "Del feedback",
+    "faq": "FAQ",
+    "privacy-policy": "Privatlivspolitik",
+    "legal-notice": "Juridisk meddelelse"
+  },
+  "mobile-main-menu": {
+    "map": "Electricity Maps",
+    "areas": "Områder",
```

Comment:
Jeg ved ikke om områder, zoner eller regioner er det bedste ord.

Reply:
When we decided on the best word, the rest of the file should be checked, because I think I have used a mix of områder and zone right now

Reply:
Should we go with område then?

Reply:
Let's do that for now yeah :)

Reply:
Wait now I got a bit confused. Should we say områder where we would say areas in english and then zone in danish for zone in english? Or områder for both areas and zone?

Reply:
@madsnedergaard what do you say?

Reply:
Sorry that I never got back on this, I'm fine with either as this is generally so much better than what we had before 😍